src/script/user/UserRepository.ts

```diff
@@ -235,9 +235,9 @@ export class UserRepository extends TypedEventEmitter<Events> {
     const {found, failed} = await this.fetchRawUsers(usersToFetch, selfUser.domain);
 
     const userWithAvailability = found.map(user => {
-      const availability = incompleteUsers
-        .concat(nonQualifiedUsers)
-        .find(incompleteUser => incompleteUser.id === user.id);
+      const availability = [...localUsers, ...incompleteUsers, ...nonQualifiedUsers].find(
```

Comment:
```suggestion
      const availability = [...dbUsers, ...nonQualifiedUsers].find(
```

Even simpler because `dbUsers = localUsers + incompleteUsers` ;)

Reply:
Addresed there: https://github.com/wireapp/wire-webapp/pull/17307/commits/8cb8900571cf3f73ee6adba5ea27d45016c9260e :)